### DISCUSSION TITLE
Disable rules from terminal

### DISCUSSION THREAD

**Discussion by intsec** [Security-Onion-Solutions/securityonion#14055] [2.4]
### Version

2.4.110

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

8

### RAM

32

### Storage for /

200G

### Storage for /nsm

320G

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

No, there are no failures

### Logs

Yes, there are additional clues in /opt/so/log/ (please provide detail below)

### Detail

I have turned on all the alert rules for my SO and now I can't get elastalert to start, it shows it is missing.  I can't disable them from the web interface since I receive an error window at the top of the page.  When ever I reboot the server is horribly slow and I really believe it's because of the alerts.  Once it stops being slow, usually a day, I can start moving around the manager in a terminal.  Where can I disable these rules?  I have gone in the the so/rules directory and have removed the rules from there but they keep coming back.

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by defensivedepth** [comment]
`I can't disable them from the web interface since I receive an error window at the top of the page. `

Please post the error so that we can troubleshoot further.

**Comment by intsec** [comment]
This is the error popup.
***@***.***

**Comment by intsec** [comment]
![image](https://github.com/user-attachments/assets/e788452b-8977-4667-ad4a-9459b887ade0)
Looks like I can't email reply with a screenshot.  Here is the error.
